dawg_python/dawgs.py

```diff
@@ -279,15 +312,14 @@ def iterkeys(self, prefix=""):
             yield u_key
 
     def items(self, prefix=""):
+        index = self.dct.ROOT
         if not isinstance(prefix, bytes):
             prefix = prefix.encode('utf8')
-        res = []
-
-        index = self.dct.ROOT
         if prefix:
             index = self.dct.follow_bytes(prefix, index)
             if not index:
-                return res
+                return
```

Comment:
this is backwards incompatible - .items should return an empty list, not None here